spec/autocomplete-manager-integration-spec.coffee

```diff
@@ -702,6 +702,38 @@ describe 'Autocomplete Manager', ->
           for item, index in editorView.querySelectorAll('.autocomplete-plus li span.word')
             expect(item.innerText).toEqual(suggestions[index])
 
+      it 'should show the suggestion list when the suppression list does not match', ->
```

Comment:
Can you move these specs under the `when an external provider is registered` section. I'm trying to reduce dependence on behavior of the default provider.